cmd/operator/main.go

```diff
@@ -34,7 +34,8 @@ import (
 )
 
 const (
-	NamespaceEnvVar string = "NAMESPACE"
+	NamespaceEnvVar         string = "NAMESPACE"
+	OpenshiftVersionsEnvVar string = "OPENSHIFT_VERSIONS"
```

Comment:
perhaps we could move this const to `internal/controller/controllers/images.go` so we can use it in both places.